### ISSUE TITLE
Make the compiler descriptions more objective

### ISSUE THREAD

**Issue by Beliavsky** [fortran-lang/fortran-lang.org#238]
Some of the compiler descriptions read like advertising copy, because they were taken from advertising. For example,

"Absoft
Our compilers build faster code more efficiently than ever before."

The "our" should be reworded since the fortran-lang site is not owned by a vendor.

The Lahey description has two exclamation points that don't below in a vendor-neutral site.

In general, vendors will try to summarize the special features of their compilers in a few sentences, and such descriptions can be informative, but I think fortran-lang should tone them down a little.

**Comment by vmagnin** [comment]
I agree with @Beliavsky and the commercial style was one of the reason it was difficult to translate that page in French.

**Comment by certik** [comment]
In order to make this section fair, I still like this idea the most: give each vendor a space of 100 words or less to put anything they want for their compilers. That way they can advertise as they wish. We can perhaps put a paragraph at the very top explaining this.

The alternative is to start editing the paragraph for each compiler, but I don't have access to all of them, and I don't want to be in a position either to be the "editor" for how compilers are described and advertised. 

By offloading this responsibility to the compiler vendors themselves, and just provide a venue and "rules" (100 words or less), and make those rules clear (currently it is not clear) seems better and ultimately fair I think.

CC @milancurcic.
